uiowa/uiowa
# Admissions AoS requirements

Resolves #2942

## To Test

Pull and go to http://admissions.local.drupal.uiowa.edu/node/761/edit
See the new requirement collections (first year, transfer, international). Fill em out. Try to break it by not adding all of the values.
On render in the same background color as the Admissions Requirements text, see some white card styles with the intro content and some buttons. If references to content weren't provided the button won't show. If a label is not provided a fall back for the button text displays, the buttons themselves should link to the content your referenced.
The card titles themselves should match the field you added them to.
Final styling will be captured in https://github.com/uiowa/uiowa/issues/2934
